src/AstarteManager.tsx

```diff
@@ -18,9 +18,11 @@
 
 import React, { createContext, useCallback, useContext, useMemo, useState } from 'react';
 import AstarteClient, { AstarteToken } from 'astarte-client';
+
 import _ from 'lodash';
 
 import type { DashboardConfig } from './types';
+import { Cookies, useCookies } from 'react-cookie';
```

Comment:
looks like `react-cookie` only allows usage with hook functions, which forces us pass `setCookie` and `removeCookie` to the `saveSession`.
What about `js-cookie`?